server/models/pages.js

```diff
@@ -926,6 +927,29 @@ module.exports = class Page extends Model {
     return rebuildJob.finished
   }
 
+  /**
+   *
+   * @returns
+   */
+  static async buildXMLSitemap() {
+    const xmlTree = [
+      '<?xml version="1.0" encoding="UTF-8"?>',
+      '<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">'
+    ]
+
+    const pages = await WIKI.models.pages.query().select(['path', 'updatedAt']).where('isPublished', '=', true)
+    const base = `${WIKI.config.host}/${WIKI.config.lang.code}`
+
+    pages.forEach(page => {
+      const date = moment(page.updatedAt).format('YYYY-MM-DDThh:mm:ssZ')
```

Comment:
No need to load moment to output an ISO date string. Date has a native [toISOString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) method.

Reply:
I lose timezone with native method, any suggestion?

Reply:
toISOString() has no timezone offset, any suggestion?

Reply:
Indeed. Leave moment then. It will be replaced by luxon in v3.